<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>786</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="EqnLabel">
        <property name="text">
         <string>Fit Function:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EqnText">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>lambda x,y,z: x+y+z</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="EqnValidBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="EnablePtsBtn">
        <property name="text">
         <string>Enable Points</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="DisablePtsBtn">
        <property name="text">
         <string>Disable Points</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTabWidget" name="verticalTabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="verticalTabWidgetPage1">
         <attribute name="title">
          <string>Data</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="DataTable" name="DataTable">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="cornerButtonEnabled">
             <bool>false</bool>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>X Data</string>
             </property>
             <property name="textAlignment">
              <set>AlignCenter</set>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Y Data</string>
             </property>
             <property name="textAlignment">
              <set>AlignCenter</set>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Variables</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Variable:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="VariableSelector">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Fitting Parameters</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Init. Guess</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="InitGuessBox">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="acceptDrops">
                   <bool>false</bool>
                  </property>
                  <property name="frame">
                   <bool>true</bool>
                  </property>
                  <property name="readOnly">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Lower Bound</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="LowerBoundBox">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="acceptDrops">
                   <bool>false</bool>
                  </property>
                  <property name="readOnly">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Upper Bound</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="UpperBoundBox">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="acceptDrops">
                   <bool>false</bool>
                  </property>
                  <property name="readOnly">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Fit Results</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Value:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="FitValueBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Lower 95%:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="LowerConfBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Upper 95%:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="UpperConfBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="MPLWidget" name="MPLWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="mplvlayout"/>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="LogTextView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>300</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MPLWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
   <slots>
    <slot>EnablePointsClicked()</slot>
    <slot>dataUpdated(PyQt_PyObject)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>DataTable</class>
   <extends>QTableWidget</extends>
   <header>datatable.h</header>
   <slots>
    <signal>dataDropped(PyQt_PyObject)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>EnablePtsBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>EnablePointsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EqnText</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>fitfuncChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DisablePtsBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>DisablePointsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>InitGuessBox</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>FitParamsChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LowerBoundBox</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>FitParamsChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UpperBoundBox</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>FitParamsChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VariableSelector</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>VariableSelectorChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>receiveData(PyQt_PyObject)</slot>
  <slot>EnablePointsClicked()</slot>
  <slot>fitfuncChanged(QString)</slot>
  <slot>DisablePointsClicked()</slot>
  <slot>FitParamsChanged(QString)</slot>
  <slot>VariableSelectorChanged(QString)</slot>
 </slots>
</ui>
